main/src/library/Set.flix

```diff
@@ -281,7 +281,7 @@ namespace Set {
 
     ///
     /// Returns the association set `xs` as a map.
-    /// If `xs` contains multiple values for the same key, the value appearing latest in `xs` is used.
+    /// If `xs` contains multiple values for the same key, the value appearing earliest in `xs` is used.
```

Comment:
I see what you mean. I was confused. I think the right answer is that: ***We will not guarantee anything if the same key appears twice!***

So instead we should just write that...: If the set contains multiple mappings for the same key, then this function does not make any guarantees about which one will be in the resulting map... or similar language.